Makefile

```diff
@@ -158,18 +154,14 @@ test_offline: check_venv check_build_reqs
 
 # The auto-deployment test needs the docker appliance
 test: check_venv check_build_reqs docker
-	TOIL_APPLIANCE_SELF=$(docker_registry)/$(docker_base_name):$(docker_tag) \
-	TRAVIS=true \
```

Comment:
If you drop TRAVIS=true, then running make test locally skips a bunch of the tests that we dropped from Gitlab by annotating them as Travis-only.

We probably need to change the travis annotation to something like not-Gitlab, so it is clear that we still want them to run when running tests manually.

Reply:
I agree with that.  I was trying to think of a better way of doing it, but like your suggestion.  You're right, I didn't expect a review yet.  I'll add the WIP.